bot/cogs/superstarify.py

```diff
@@ -145,11 +149,14 @@ def moderation(self) -> Moderation:
 
             # Log to the mod_log channel
             log.trace("Logging to the #mod-log channel. This could fail because of channel permissions.")
-            mod_log = self.bot.get_channel(Channels.modlog)
-            await mod_log.send(
-                f":middle_finger: {member.name}#{member.discriminator} (`{member.id}`) "
-                f"has been superstarified by **{ctx.author.name}**. Their new nickname is `{forced_nick}`. "
-                f"They will not be able to change their nickname again until **{end_time}**"
+            mod_log_message = (
+                f"{member.name}#{member.discriminator} (`{member.id}`)\n\n"
+                f"Superstarified by **{ctx.author.name}**\n"
+                f"New nickname:`{forced_nick}`\n"
+                f"Superstardom ends: **{end_time}**"
+            )
+            await self.modlog.send_log_message(
```

Comment:
This now works fine and works locally, though maybe it'd be nice to have the thumbnail be the user avatar (as is done with other logs). `send_log_message` accepts a `thumbnail` kwarg so you can make use of that.